### ISSUE TITLE
A problem was encountered during the testing

### ISSUE THREAD

**Issue by zem118** [SarahwXU/HiSup#19]
Thank you for this great project!
I've finished training to get a training model and now I want to test it, but when I run test.py it prompts an error and I don't know what it means?
![8cfe9a078d05ee8c955080ab44c5189](https://github.com/SarahwXU/HiSup/assets/140385841/4f7d8bb8-e628-4519-b6ae-a2296388f8fb)

**Comment by XJKunnn** [comment]
You haven't install the Boundary IoU API, please read the README file carefully. You have to finish the installation step before testing.

**Comment by zem118** [comment]
I have installed the Boundary IoU API as per README
![60f545b2706d93727acaced943bf053](https://github.com/SarahwXU/HiSup/assets/140385841/5b93476c-0d4d-4ac1-b02d-9229493421be)
并且按照这些进行了修改
![dbbe8b9c237715ac15348f85d0d3a7a](https://github.com/SarahwXU/HiSup/assets/140385841/d510ed53-0255-47fc-a3fe-767f836abd86)
但是运行tese.py还是出现如下错误
![18aea1f8a04cde4ce160b1c1af103db](https://github.com/SarahwXU/HiSup/assets/140385841/bad13fb2-ff2b-4fe8-9406-563feb446faa)

**Comment by SarahwXU** [comment]
Since the problem is the installation of boundary iou library, I suggest that you individually test the code of "from boundary_iou.coco_instance_api.coco import COCO" to make sure that it works. This part has nothing to do with our model.

**Comment by lirui-liu** [comment]
> I have installed the Boundary IoU API as per README ![60f545b2706d93727acaced943bf053](https://user-images.githubusercontent.com/140385841/256973905-5b93476c-0d4d-4ac1-b02d-9229493421be.png) ，and modified it according to these ![dbbe8b9c237715ac15348f85d0d3a7a](https://user-images.githubusercontent.com/140385841/256973927-d510ed53-0255-47fc-a3fe-767f836abd86.png) ，But running tese.py still gives me the following error ![18aea1f8a04cde4ce160b1c1af103db](https://user-images.githubusercontent.com/140385841/256973934-bad13fb2-ff2b-4fe8-9406-563feb446faa.png)

you can put the dir boundary_iou in tool/

**Comment by SarahwXU** [comment]
It is recommended to install the Boundary IoU in the second way:
```
git clone git@github.com:bowenc0221/boundary-iou-api.git
cd boundary_iou_api
pip install -e .
```

**Comment by lirui-liu** [comment]
yes,my fault, it is the right way, i followed the tips and it works sucessfully!   thanks for your recommend!
then i want to ask how to load own trained  model .pth when i run demo.py ? i don't know how to  modify , please give  me a tips

**Comment by lirui-liu** [comment]
> It is recommended to install the Boundary IoU in the second way:
> 
> ```
> git clone git@github.com:bowenc0221/boundary-iou-api.git
> cd boundary_iou_api
> pip install -e .
> ```
yes,my fault, it is the right way, i followed the tips and it worked sucessfully!   thanks for your recommend!
then i want to ask how to load own trained  model .pth when i run demo.py ? i don't know how to  modify , please give  me a tips

**Comment by SarahwXU** [comment]
The function named "get_pretrained_model" is responsible for loading pretrained parameters in demo.py. The implementation is in https://github.com/SarahwXU/HiSup/blob/69e0cb3c7039bd038139ae0b95ea4ef03b2e5162/hisup/detector.py#L228C45-L228C45, which you may use as a simple example. If you want to test your own model, it is highly recommended to write your personal testing code following the test.py in the "scripts" file.
